api/app/signals/apps/signals/models/expression.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.gis.db import models
+
+
+class ExpressionType(models.Model):
+    class Meta:
+        verbose_name = 'ExpressionsType'
+
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(max_length=3000)
+
+    def __str__(self):
+        return self.name
+
+
+class ExpressionContext(models.Model):
```

Comment:
Consider whether ExpressionContext instances can be generated from the Context implemenations like the SignalContext in this PR.

Concern is that the Python implementation of a <Somthing>Context may deviate from what is in the ExpressionContext table. (A rule could validate and not run, because there are two sources of truth).

Reply:
Expression endpoint should be independent of concrete implementations such as routing. We need to maintain the expression context in order to perform validation of expression. I will add logic in the 2nd part of the PR that a rule only will be applied when it's validated

Reply:
We discussed this Thursday 2020-10-08, the ExpressionContext table is not user editable --- it is filled by the developers implementing new `<Something>Context` classes.